docroot/themes/humsci/humsci_basic/src/scss/components/_main-nav.scss

```diff
@@ -1,105 +1,363 @@
+        }
+      }
+    }
+
+    &:hover:not(.is-active) {
+      background-color: $hb-menu-link-background-hover;
+    }
+
+    // Targeting only the first level of navigation on desktop
+    .hb-main-nav__menu-lv1 > li > & {
+      @include grid-media-min('lg') {
+        .js &::before {
+          content: '';
+          display: block;
+          position: absolute;
+          width: 100%;
+          height: hb-calculate-rems(4px);
+          bottom: hb-calculate-rems(8px);
+          top: auto;
+          left: 0;
+          background: transparent;
+        }
+
+        .js &.is-active {
+          background-color: transparent;
+        }
+
+        .js &.hb-main-nav__link--active-trail::before {
+          @include hb-colorful {
+            background: hb-colorful-variation(secondary);
+          }
+        }
+
+        .js &:hover {
+          background-color: transparent;
+
+          &:not(.is-active):not(.hb-main-nav__link--active-trail)::before {
+            background: $hb-menu-link-border-hover;
+          }
+        }
+      }
+    }
+
+    .hb-main-nav__menu-lv2 & {
+      @include grid-media-min('lg') {
+        .js & {
+          font-size: hb-calculate-rems(16px);
+          padding: hb-calculate-rems(10px) hb-calculate-rems(14px);
+        }
+      }
+    }
+
+    &--active-trail {
+      &::before {
+        content: '';
+        display: block;
+        width: hb-calculate-rems(5px);
+        position: absolute;
+        top: 0;
+        bottom: 0;
+        left: 0;
+
+        background: $hb-menu-link-border-active;
+      }
+
+      &.is-active {
+        @include hb-colorful {
+          background: rgba(darken(hb-colorful-variation(secondary), 2%), 0.2);
+
+          &::before {
+            background: hb-colorful-variation(secondary);
+          }
+        }
+      }
+    }
+
+    &.hb-nested-toggler {
+      position: relative;
+
+      &::after {
+        @include hb-menu-icon-wrapper(48px, 48px);
+        @include hb-menu-plus-icon;
+        content: '';
+        display: block;
+
+        position: absolute;
+        top: hb-calculate-rems(8px);
+        right: 0;
+
+        // If there is no JS then the navigation is automatically expanded and
+        // toggle buttons are unnecessary.
+        .no-js & {
+          display: none;
+        }
+
+        @include grid-media-min('md') {
+          top: hb-calculate-rems(16px);
+        }
+
+        @include grid-media-min('lg') {
+          .js & {
+            @include hb-menu-icon-wrapper(20px, 15px);
+            @include hb-menu-down-arrow-icon;
+
+            bottom: hb-calculate-rems(20px);
+            top: auto;
+          }
+        }
+      }
+
+
+      &[aria-expanded="true"]::after {
+        @include hb-menu-minus-icon;
+
+        @include grid-media-min('lg') {
+          .js & {
+            @include hb-menu-up-arrow-icon;
+          }
+        }
+      }
     }
   }
 
   &__button {
```

Comment:
is there any way to consolidate these styles with those on the nested toggler? they seem pretty similar

Reply:
maybe make a plus/minus-icon-menu mixin?

Reply:
I tried but since one is a button and the other is a pseudo element they have a different selector for when they are open (hope that makes sense. The also have some different rules for how the icons are absolutely positioned. The mixins were created to keep things consistent.

Reply:
ok